npm/test/basics.js

```diff
@@ -280,6 +280,39 @@ test("all katas work", async () => {
   );
 });
 
+test("IDs used in katas are unique", async () => {
```

Comment:
I'd kind of expect this to be part of `generateKatas.js` so that we can fail quickly at build time. Would that be easy?

Reply:
I considered that option too, but my initial thought was that it would be "safer" to check using the APIs that the frontend uses. At the end, it would use the same JS objects, so I guess it makes sense to move this check at build time. I'll change it.

Reply:
Updated.